pkg/istio-agent/xds_proxy.go

```diff
@@ -247,11 +285,17 @@ func (p *XdsProxy) StreamAggregatedResources(downstream discovery.AggregatedDisc
 			con.requestsChan <- req
 			if !initialRequestsSent && req.TypeUrl == v3.ListenerType {
 				// fire off an initial NDS request
```

Comment:
s/NDS/DNS/ - and probably change the comment ( 'fire off requests for types handled by proxy, like DNS and ProxyConfig')

Reply:
NDS is nametable discovery service - confusingly similar to "DNS"